spinalcordtoolbox/deepseg/models.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+"""
+Deals with models for deepseg module. Available models are listed under MODELS.
+"""
+
+
+import os
+import logging
+
+from spinalcordtoolbox import __sct_dir__
+
+logger = logging.getLogger(__name__)
+
+# List of models. The convention for model names is: (species)_(university)_(contrast)_region
+# Regions could be: sc, gm, lesion, tumor
+MODELS = {
+    't2star_sc':
+        {'url': 'https://osf.io/v9hs8/download?version=2',
+         'description': 'Cord segmentation on T2*-weighted contrast.'},
+    'uqueensland_mice_sc':
+        {'url': 'https://osf.io/nu3ma/download?version=2',
+         'description': 'Cord segmentation on mouse MRI. Data from University of Queensland.'},
+    'uqueensland_mice_gm':
```

Comment:
According to the convention this should be

```suggestion
    'mice_uqueensland_gm':
```

Reply:
fixed in 57b99e394347ad4383ecbd7221234b6a6c9f3138